### PR TITLE
Add support for columns order in query (Column attribute).

### PR DESCRIPTION
**Pull request by ConstantinOr**
### Issues

*This pull request fixes issue #1085.*

### Description

*Briefly describe the changes of this pull request.*

### Checklist (Uncheck if it is not completed)

- [ x ] Test cases added
- [ x ] Build and test with one-click build and test script passed

### Additional work necessary

*If documentation update is needed, please add "Docs Needed" label to the issue and provide details about the required document change in the issue.*

### CODE REVIEW

File: src/System.Web.OData/OData/Builder/StructuralTypeConfiguration.cs [hunk 1]
```diff
@@ -211,9 +211,27 @@ public virtual IEnumerable<NavigationPropertyConfiguration> NavigationProperties
...
+            {
+                if (explicitInnerProperties.Any(a => a.Value.QueryConfiguration != null &&
+                        a.Value.QueryConfiguration.ModelBoundQuerySettings != null))
+                {
+                    explicitInnerProperties = OrderColumn(explicitInnerProperties);
+                }
+
+                return explicitInnerProperties;
+            }
+            private set => explicitInnerProperties = value;
```
Comment (biaol-odata) [inline src/System.Web.OData/OData/Builder/StructuralTypeConfiguration.cs]: change to this?
private set { explicitInnerProperties = value; }

Comment (ysmoradi) [reply src/System.Web.OData/OData/Builder/StructuralTypeConfiguration.cs]: Shall we set C# version in csproj, instead of these round trips?

Comment (ConstantinOr) [reply src/System.Web.OData/OData/Builder/StructuralTypeConfiguration.cs]: Hello. I  can fix property and resubmit pull request. What you can recommend?

Comment (biaol-odata) [reply src/System.Web.OData/OData/Builder/StructuralTypeConfiguration.cs]: @ConstantinOr Can you please also add some tests to show that the issue is fixed?

Comment (ConstantinOr) [reply src/System.Web.OData/OData/Builder/StructuralTypeConfiguration.cs]: Ok. No problem.

**Comment by ConstantinOr** [comment]
Sorry i was incorrectly understood problem.

**Pull request closed without merging**